builtins/web/timers.cpp

```diff
@@ -113,7 +117,7 @@ class TimerTask final : public api::AsyncTask {
   }
 };
 
-std::map<int32_t, PollableHandle> TimerTask::timer_ids_ = {};
```

Comment:
As-is, this isn't GC-safe: if a moving GC were to happen and nothing else were to update the references inside `TimerTask`, they would become invalid.

In practice this situation can probably not currently arise, because any task included here should also be in the event queue. But that seems dangerous to rely on in the face of potential refactorings later on.

One way I can see to make this work is to introduce a new class, `TimersMap` which has a `trace` method and holds the `timer_ids_` field. We can then add a singleton instance of that as a `static PersistentRooted<TimersMap>` member of `TimerTask` and have the `trace` method iterate over the map to trace through the contained tasks.